tests/unit/MotionAnimatorTests.swift

```diff
@@ -45,7 +45,7 @@ class MotionAnimatorTests: XCTestCase {
 
   func testCompletionCallbackIsExecutedWithZeroDuration() {
     let animator = MotionAnimator()
-    let traits = MDMAnimationTraits(duration: 1)
+    let traits = MDMAnimationTraits(duration: 0)
```

Comment:
Is this a "real zero" or will it fall back to CoreAnimation's 0.250 seconds?

Reply:
The Motion Animator supports "real zero" durations, so this will not fall back to the Core Animation default.